docs/build.md

```diff
@@ -51,6 +51,7 @@ The `Build` definition supports the following fields:
   - `spec.resources` - Refers to the compute [resources](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/) used on the container where the image is built.
   - `spec.parameters` - Refers to a list of `name-value` that could be used to loosely type parameters in the `BuildStrategy`.
   - `spec.dockerfile` - Path to a Dockerfile to be used for building an image. (_Use this path for strategies that require a Dockerfile_)
+  - `spec.timeout` - Defines a custom timeout. The value needs to be parsable by [ParseDuration](https://golang.org/pkg/time/#ParseDuration), for example `5m`. The default is ten minutes. Can be overwritten in the `BuildRun`.
```

Comment:
Shouldn't this be - `Can be overwritten in the Build` instead of `Can be overwritten in the BuildRun.` considering the file is `docs/build.md`

Reply:
No, I did propably not the best wording. But the order is:

1. Take the timeout from the buildrun
2. Take the timeout from the build
3. Take the default timeout (which comes from Tekton)

Reply:
@DhritiShikhar let's discuss there: https://github.com/redhat-developer/build/pull/201